### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by akosyakov**
This PR updates dependencies in the lock file and fixes the follow-up issues. It fixes as well #64 and #39.

### CODE REVIEW

File: src/converter.ts [hunk 1]
```diff
@@ -139,6 +140,24 @@ export class MonacoToProtocolConverter {
         }
     }
 
+    asTextEdit(edit: monaco.editor.ISingleEditOperation): TextEdit {
```
Comment (rcjsuen) [inline src/converter.ts]: Should name this `asISingleEditOperation` instead.

File: src/converter.ts [hunk 2]
```diff
@@ -139,6 +140,24 @@ export class MonacoToProtocolConverter {
...
+        return {
+            range,
+            newText: edit.text
+        }
+    }
+
+    asTextEdits(items: monaco.editor.ISingleEditOperation[]): TextEdit[];
+    asTextEdits(items: undefined | null): undefined;
+    asTextEdits(items: monaco.editor.ISingleEditOperation[] | undefined | null): TextEdit[] | undefined;
+    asTextEdits(items: monaco.editor.ISingleEditOperation[] | undefined | null): TextEdit[] | undefined {
```
Comment (rcjsuen) [inline src/converter.ts]: Should name these `asISingleEditOperations` instead.

File: src/converter.ts (hunk 1 above)
Comment (rcjsuen) [reply src/converter.ts]: Sorry, ignore this. I read the signatures wrong.

File: src/converter.ts (hunk 2 above)
Comment (rcjsuen) [reply src/converter.ts]: Sorry, ignore this. I read the signatures wrong.

**Comment by akosyakov** [comment]
@rcjsuen thanks for looking at it. It was not your change, dependencies have not been updated for a while. I've pushed again to address your comments.

**Pull request merged**